lib/flatpak-game.lua

```diff
@@ -351,7 +351,7 @@ function save_manifest(metadata, enable_network)
 	-- http://flatpak.org/developer.html#Anatomy_of_a_Flatpak_App
 	write_line(f, '[Application]')
 	write_line(f, 'name=' .. metadata.id)
-	write_line(f, 'runtime=org.freedesktop.Platform/' .. metadata.arch .. '/1.4')
+	write_line(f, 'runtime=org.freedesktop.Platform/' .. metadata.arch .. '/1.6')
```

Comment:
Yeah, that wasn't the case when I did that pull request. I was trying to reduce number of runtimes installed locally, nothing else really. Will close the PR…